sci-libs/rdkit/rdkit-2024.03.4.ebuild

```diff
@@ -54,5 +53,6 @@ src_configure() {
 }
 
 src_test() {
-	RDBASE="${WORKDIR}/${PN}-Release_2021_09_4_build" cmake_src_test
+	rsync -aur "${S}_build/rdkit" "${S}" || die
```

Comment:
I'm not sure about the rsync here, but it's the only way I could get the python import test to work

Reply:
Instead of copying the build stuff into the `PYTHONPATH`, simply set `PYTHONPATH` to where the built stuff is.

Reply:
The problem here seems to be that the build directory contains all `.so` files but no `.py` files. So setting the `PYTHONPATH` to the build directory will produce an error that it's not a package

Reply:
Does `CMAKE_IN_SOURCE_BUILD=1` help?

Reply:
`CMAKE_IN_SOURCE_BUILD=1` did indeed solve it